dask_sql/context.py

```diff
@@ -246,12 +251,25 @@ def create_table(
         )
 
         self.schema[schema_name].tables[table_name.lower()] = dc
+
         if statistics:
             self.schema[schema_name].statistics[table_name.lower()] = statistics
+        elif parquet_statistics:
+            statistics = parquet_statistics(dc.df)
+            if statistics:
+                row_count = 0
+                for d in statistics:
+                    row_count += d["num-rows"]
+                statistics = Statistics(row_count)
+                self.schema[schema_name].statistics[table_name.lower()] = statistics
+
+        # If no statistics are obtainable, we will just assume 100 rows
+        if not statistics:
```

Comment:
```suggestion
        else:
```

Reply:
I originally had `else`, but I don't think that would cover all cases. If we get into the `elif` block (meaning that `parquet_statistics()` is importable) but nothing is returned by `parquet_statistics()`, then a value for `statistics` would never be assigned and fail a couple lines later with `statistics.row_count`.

We could do 2 `else` blocks (one associated with the inner `if statistics` and the other with the outer `if statistics`/`elif parquet_statistics`) but this way is less lines of code.

Reply:
Thanks for the explanation that makes sense to me!